samza-api/src/main/java/org/apache/samza/runtime/ApplicationRunner.java

```diff
@@ -98,5 +104,5 @@ static ApplicationRunner fromConfig(Config config) {
    * @param streamId  The logical identifier for the stream in Samza.
    * @return          The {@link StreamSpec} instance.
    */
-  StreamSpec streamFromConfig(String streamId);
+  public abstract StreamSpec getStream(String streamId);
```

Comment:
@jmakes I think it'll be clearer to call this getStreamSpec to disambiguate b/w all the uses of Stream in fluent API (I'm thinking logical stream creation in StreamGraph and physical stream creation in ExecutionPlanner). What do you think?